draft-irtf-cfrg-rsa-blind-signatures.md

```diff
@@ -245,6 +245,16 @@ Steps:
 11. output blinded_msg, inv
 ~~~
 
+The blinding factor r must be randomly chosen from a uniform distribution.
+
+In order to do so, the random_integer_uniform function can be implemented by obtaining L uniform bytes, where
+
+~~~
+L = ceil((ceil(log2(n - m)) + k) / 8)
```

Comment:
Does this heuristic come from somewhere we can reference? If not, we might want to add a line or two about how this was defined.

Reply:
It was included in [FIPS-186](https://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.186-4.pdf), though not in this general form. We could just cite that?